src/GitLink/Program.cs

```diff
@@ -24,51 +27,51 @@ private static int Main(string[] args)
             var consoleLogListener = new OutputLogListener();
             LogManager.AddListener(consoleLogListener);
 
-            try
+            Uri remoteGitUrl = null;
+            string pdbPath = null;
+            bool skipVerify = false;
+            bool downloadWithPowershell = false;
+            var arguments = ArgumentSyntax.Parse(args, syntax =>
             {
-                HelpWriter.WriteAppHeader(s => Log.Write(LogEvent.Info, s));
+                syntax.DefineOption("u|url", ref remoteGitUrl, s => new Uri(s, UriKind.Absolute), "Url to remote git repository.");
```

Comment:
Why? 

b|branch doesn't mean anything as we always use a commit since that ensures the code downloaded matches what was built. Although if someone for some bizarre reason prefers to use a branch instead of a commit ID, they are free to do so with the `--commit` parameter since all that matters is that it's a 'commit-ish', as I think the terminology goes. But without any of these parameters, we use HEAD from the git repo. Distinguishing between branch and shaHash doesn't seem to make any sense. But if I'm missing something, please share.

Also, the command line is changing anyway. It doesn't take config or platform or sln any more, so it's not like there's any good in trying to preserve command line backward compatibility.

I think I lost file logging and "errorsaswarnings" along the way though. So here's to track that:

* [ ] Restore errorsaswarnings
* [ ] Restore file logging